doc.go

```diff
@@ -1,129 +1,75 @@
-the `X-Rate-Limit-Reset` header as the amount of seconds to wait.
-
-# Configuration
-
-There are several other options that can be specified during the creation of a
-new client.
-
-	m, err := management.New(
-		domain,
-		management.WithClientCredentials(context.Background(), id, secret),
-		management.WithDebug(true),
-	)
-
-## Request Options
-
-As with the global client configuration, fine-grained configuration can be done
-on a request basis.
-
-	c, err := m.Connection.List(
-		context.Background(),
-		management.Page(2),
-		management.PerPage(10),
-		management.IncludeFields("id", "name", "options"),
-		management.Parameter("strategy", "auth0"),
-	)
-*/
+// Package auth0 is the Auth0 SDK for Go.
+//
+// The SDK provides clients that interact with the Auth0 Authentication and Management APIs
+// and is split into separate packages as such.
+//
+//   - auth0 - Provides helpers for providing values as pointers.
+//   - authentication - Provides an Authentication Client for use when interacting with the
+//     [Authentication API].
+//   - management - Provides a Management Client for use when interacting with the Auth0
+//     [Management API].
+//
+// # Getting Started
+//
+// Install the SDK using `go get`
+//
+//	go get github.com/auth0/go-auth0
+//
+// # Authentication
+//
+// Below is an example of using the Authentication client, for full documentation visit the
+// [authentication client docs].
+//
+//	authAPI, err := authentication.New(
+//		context.Background(),
+//		domain,
+//		authentication.WithClientID(id),
+//		authentication.WithClientSecret(secret), // Optional depending on the grants used
+//	)
+//	if err != nil {
+//		// handle err
+//	}
+//	// Now we have an authentication client, we can interact with the Auth0 Authentication API.
+//	// Sign up a user
+//	userData := database.SignupRequest{
+//		Connection: "Username-Password-Authentication",
+//		Username:   "mytestaccount",
+//		Password:   "mypassword",
+//		Email:      "mytestaccount@example.com",
+//	}
+//	createdUser, err := authAPI.Database.Signup(context.Background(), userData)
+//	if err != nil {
+//		// handle err
+//	}
+//
+// # Management
+//
+// Below is an example of using the Management client, for full documentation visit the
+// [management client docs].
+//
+//	import (
+//		"github.com/auth0/go-auth0"
+//		"github.com/auth0/go-auth0/management"
+//	)
+//	// Initialize a new client using a domain, client ID and secret.
+//	m, err := management.New(
+//		domain,
+//		management.WithClientCredentials(context.Background(), id, secret),
+//	)
+//	if err != nil {
+//		// handle err
+//	}
+//	c := &management.Client{
+//		Name:        auth0.String("Client Name"),
+//		Description: auth0.String("Long description of client"),
+//	}
+//	err = m.Client.Create(context.Background(), c)
+//	if err != nil {
+//		// handle err
+//	}
+//
+// [management client docs]: https://pkg.go.dev/github.com/auth0/go-auth0/management
```

Comment:
I'm not super thrilled about having to link to pkg.go.dev directly but I'm not sure the [doc links](https://go.dev/doc/comment#links:~:text=in%20HTML%20renderings.-,Doc%20links,-Doc%20links%20are) support linking like I want to here (i.e to the top level docs)

Reply:
Would a relative link work?

<img width="869" alt="Screenshot 2023-11-10 at 13 36 29" src="https://github.com/auth0/go-auth0/assets/5055789/632660ef-f3d3-4a39-b8cd-b9f81d77800b">

Reply:
Unfortunately not, we can't use HTML in the docs and the URL support only seems to support full URLs rather than relative.